<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EyeModel</class>
 <widget class="QWidget" name="EyeModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>469</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <hintingpreference>PreferDefaultHinting</hintingpreference>
    <fontweight>Medium</fontweight>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="ImageEye">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>471</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>491</width>
     <height>241</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../assets/halfeyeimage2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Fovea">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Fovea</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="VitreousBody">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Vitreous body</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="CrystallineLens">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Crystaline 
Lens</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="CilliaryBody">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ciliary Body</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Retina">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Retina</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Choroid">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Choroid</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Sclera">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Sclera</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="OpticNerve">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Optic Nerve</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="RetinalArteryAndVein">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Central Retinal 
Artery and Vein</string>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoRepeatInterval">
    <number>98</number>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextOnly</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="AgueousHumor">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Aqueous 
Humor</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="PosteriorChamber">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Posterior 
Chamber</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Conjuctiva">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Conjuctiva</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="SuspensoryLigament">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Suspensory 
Ligament</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="OpticDisk">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Optic Disk</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Iris">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Iris</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Pupil">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Pupil</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="AnteriorChamber">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Anterior 
Chamber</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="OraSerrata">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ora Serrata</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="Macula">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Macula</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="HyaloidCanal">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orbitron</family>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Hyaloid 
Canal</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
